[2025-01-08T19:40:13.746-0800] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-08T19:40:13.758-0800] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: CovidE2EPipeDag.insert_staged_raw_data manual__2025-01-09T03:38:16.240891+00:00 [queued]>
[2025-01-08T19:40:13.768-0800] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: CovidE2EPipeDag.insert_staged_raw_data manual__2025-01-09T03:38:16.240891+00:00 [queued]>
[2025-01-08T19:40:13.768-0800] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-01-08T19:40:13.783-0800] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): insert_staged_raw_data> on 2025-01-09 03:38:16.240891+00:00
[2025-01-08T19:40:13.786-0800] {standard_task_runner.py:72} INFO - Started process 10119 to run task
[2025-01-08T19:40:13.794-0800] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'CovidE2EPipeDag', 'insert_staged_raw_data', 'manual__2025-01-09T03:38:16.240891+00:00', '--job-id', '1692', '--raw', '--subdir', 'DAGS_FOLDER/CovidE2EPipeDag.py', '--cfg-path', '/var/folders/vm/rn9v1g1j739_mq8j5pfl1yxm0000gn/T/tmpbsngedfy']
[2025-01-08T19:40:13.797-0800] {standard_task_runner.py:105} INFO - Job 1692: Subtask insert_staged_raw_data
[2025-01-08T19:40:13.850-0800] {task_command.py:467} INFO - Running <TaskInstance: CovidE2EPipeDag.insert_staged_raw_data manual__2025-01-09T03:38:16.240891+00:00 [running]> on host Matthews-MacBook-Air.local
[2025-01-08T19:40:13.949-0800] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Matthew Horn' AIRFLOW_CTX_DAG_ID='CovidE2EPipeDag' AIRFLOW_CTX_TASK_ID='insert_staged_raw_data' AIRFLOW_CTX_EXECUTION_DATE='2025-01-09T03:38:16.240891+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-09T03:38:16.240891+00:00'
[2025-01-08T19:40:13.951-0800] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-08T19:40:13.995-0800] {subprocess.py:78} INFO - Tmp dir root location: /var/folders/vm/rn9v1g1j739_mq8j5pfl1yxm0000gn/T
[2025-01-08T19:40:13.997-0800] {subprocess.py:88} INFO - Running command: ['/bin/bash', '-c', 'snow stage execute @CovidE2EPipeDatabase.raw_data.raw_data_stage/load_staged_data_2025-01-08.sql']
[2025-01-08T19:40:14.005-0800] {subprocess.py:99} INFO - Output:
[2025-01-08T19:40:20.942-0800] {subprocess.py:106} INFO - SUCCESS - @CovidE2EPipeDatabase.raw_data.raw_data_stage/load_staged_data_2025-01-08.sql
[2025-01-08T19:40:20.947-0800] {subprocess.py:106} INFO - +---------------------------------------------------+
[2025-01-08T19:40:20.950-0800] {subprocess.py:106} INFO - | File                            | Status  | Error |
[2025-01-08T19:40:20.950-0800] {subprocess.py:106} INFO - |---------------------------------+---------+-------|
[2025-01-08T19:40:20.951-0800] {subprocess.py:106} INFO - | @CovidE2EPipeDatabase.raw_data. | SUCCESS | None  |
[2025-01-08T19:40:20.951-0800] {subprocess.py:106} INFO - | raw_data_stage/load_staged_data |         |       |
[2025-01-08T19:40:20.952-0800] {subprocess.py:106} INFO - | _2025-01-08.sql                 |         |       |
[2025-01-08T19:40:20.953-0800] {subprocess.py:106} INFO - +---------------------------------------------------+
[2025-01-08T19:40:21.353-0800] {subprocess.py:110} INFO - Command exited with return code 0
[2025-01-08T19:40:21.378-0800] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-08T19:40:21.380-0800] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=CovidE2EPipeDag, task_id=insert_staged_raw_data, run_id=manual__2025-01-09T03:38:16.240891+00:00, execution_date=20250109T033816, start_date=20250109T034013, end_date=20250109T034021
[2025-01-08T19:40:21.416-0800] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-08T19:40:21.445-0800] {taskinstance.py:3895} INFO - 3 downstream tasks scheduled from follow-on schedule check
[2025-01-08T19:40:21.449-0800] {local_task_job_runner.py:245} INFO - ::endgroup::
